CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.5 (2019-01-10)
+
+- Fixed checkbox component to not add `aria-label` to HTML tags unless a label is supplied. [#14](https://github.com/blackbaud/skyux-forms/pull/14)
```

Comment:
It might just be me, but I always find the "to not" construction awkward. I always have to read it twice to make sure I understand. I'd change the wording here to: "Fixed checkbox component to only add `aria-label` to HTML tags when it is specified." ... (Also, note that I changed "a label" to "it" to be more clear that we are talking about aria-label. That's my understanding based on your description in PR14. But if we are using "label" to refer to some other label property, let me know.)